src/Fullcount.sol

```diff
@@ -474,7 +473,7 @@ contract Fullcount is EIP712 {
         SessionAtBat[nextSessionID] = atBatID;
     }
 
-    // TODO Change name of function as tokens are no longer staked?
+    // Possibly change name of function as tokens are no longer staked?
```

Comment:
Can you retain the `TODO:` prefix. I regularly do `git grep TODO` and `git grep TODO(zomglings)`.